.clang-format

```
@@ -1,71 +1,43 @@
 BasedOnStyle: LLVM
-IndentWidth: 4
-UseTab: Never
-Language: Cpp
-Standard: Cpp11
 
 AccessModifierOffset: -4
 
-AlignConsecutiveDeclarations: false
-AlignConsecutiveAssignments: false
 AlignConsecutiveMacros: true
-AlignTrailingComments: true
+AlignConsecutiveAssignments: true
+AlignConsecutiveBitFields: true
```

```suggestion
AlignConsecutiveAssignments: false
AlignConsecutiveBitFields: false
```

it's a matter of opinion but I like the option without alignments

---

I would propose to keep `AlignConsecutiveBitFields: true`, since it make the code more readable and structed, but I will fix `AlignConsecutiveAssignments`.